docs/analytics.md

```diff
@@ -1,5 +1,9 @@
 # Analytics
 
+!!! info "Analytics platform discontinued after January 31, 2024"
+
+    The current analytics platform will be discontinued after January 31, 2024, and will be replaced by new [advanced processing functionalities](https://docs.up42.com/processing-platform/advanced). This change will affect projects, workflows, jobs, data blocks, processing blocks, and custom blocks. For more information, see the [blog post.](https://up42.com/blog/pansharpening-an-initial-view-into-our-advanced-analytic-capabilities?utm_source=documentation)
```

Comment:
[](https://docs.up42.com/processing-platform/advanced)

this URL isn't working

Reply:
I'm publishing it right now, it will go live in 2 minutes. @seedlit